cmd/edge-orchestration/main.go

```diff
@@ -108,15 +104,13 @@ func orchestrationInit() error {
 	if len(secure) > 0 {
 		if strings.Compare(strings.ToLower(secure), "true") == 0 {
 			log.Println("Orchestration init with secure option")
+			securemgr.Start(edgeDir)
```

Comment:
Is  it possible to start securemgr in the [Start() from OrchestrationBuilder](https://github.com/lf-edge/edge-home-orchestration-go/blob/8f6477f98388060ce3ce77c70eb7ab0e3cb9ad30/internal/orchestrationapi/orchestration.go#L214)?
`securemgr.Start()` is duplicated on multiple platforms and it would be better to manage it in one place. And I believe orchestrationapi package manages managers.

Reply:
@t25kim  That's a good idea, I plan to do it in the future after protected mode is used by default.